geograypher/meshes/meshes.py

```diff
@@ -124,6 +124,16 @@ def __init__(
         )
         # Load the texture
         self.logger.info("Loading texture")
+        # load IDs_to_labels
+        # if IDs_to_labels not provided, check the directory of the mesh and get the file if found
+        if IDs_to_labels is None and isinstance(mesh, PATH_TYPE.__args__):
+                possible_json = Path(Path(mesh).stem + '_IDs_to_labels.json')
```

Comment:
This indent looks like more than four spaces.

Reply:
Fixed.